python/custreamz/requirements.txt

```diff
@@ -0,0 +1,3 @@
+streamz
```

Comment:
Typically `requirements.txt` is for a pip install, which cudf doesn't support.

This file likely doesn't need to exist, and instead there should be a line in the README with the appropriate conda install command instead.

Reply:
Fixed